### ISSUE TITLE
Randomize splitting of documents

### ISSUE THREAD

**Issue by igrangel** [i40-Tools/AMLGoldStandardGenerator#20]
How can you ensure [here](https://github.com/i40-Tools/AMLGoldStandardGenerator/blob/6e56a96cf48ae7626e0170958e498f02c3b7052b/src/main/java/main/AMLGoldStandardGenerator.java#L338) that elements are randomly generated?

**Comment by omarrana** [comment]
Elements are Randomly Generated once at the start then they are just appended to seed.aml seed-0, and seed-1. The code above links to just randomly splitting few items into one of the files so they are bit more different.

**Comment by igrangel** [comment]
The idea to implement would be the following. The algorithm should randomly take elements and create conflicts, i.e., newly added elements to `seed-0` or `seed-1`. Whenever they are created should be stated in the Gold standard that those elements are 0 since they will be in one seed but not in the other.

**Comment by omarrana** [comment]
yes the new elements generation is not implemented in seed-0 or seed-1 with 0 value in GoldStandard.
So technically we dont need to tell GoldStandard they are 0 since GoldStandard takes values from seed.aml. 
We just add some new random values in seed1 and seed 0. Is it really what we are looking for

**Comment by omarrana** [comment]
close

**Comment by igrangel** [comment]
Which distribution is the basis to make this random generation? Please, point to the line of code.
